charts/bulk-scan-payment-processor/values.yaml

```diff
@@ -4,11 +4,14 @@ java:
   ingressHost: bulk-scan-payment-processor-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   environment:
     PAYMENTS_QUEUE_MAX_DELIVERY_COUNT: "5"
+    S2S_NAME: "bulk_scan_payment_processor"
+    S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   image: 'https://hmctspublic.azurecr.io/bulk-scan/payment-processor:latest'
   keyVaults:
     "bulk-scan":
       resourceGroup: bulk-scan
       secrets:
+        - microservicekey-bulk-scan-payment-processor
```

Comment:
perhaps `s2s-secret-payment-processor`?

Reply:
`microservicekey` prefix is needed for some automation in terraform for S2S. On our side we can call the secret sth like `s2s_secret` or something